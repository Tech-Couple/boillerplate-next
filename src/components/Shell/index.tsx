import React from 'react'
import styled from 'styled-components'

const Background = styled.div`
  background-color: #d4d4d4;
  height: 1000vh;
`

const Text = styled.div`
  color: #06092b;
`

const Shell = () => {
  return (
    <Background>
      <Text>
        BOILERPLATE WITH: NEXTJS, TYPESCRIPT, JEST AND REACT TESTING LIBRARY,
        STYLED COMPONENTS AND STORYBOOK.
      </Text>
    </Background>
  )
}

export default Shell
