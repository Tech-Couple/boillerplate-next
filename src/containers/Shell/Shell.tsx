import React from 'react'
import styled from 'styled-components'

const Background = styled.div`
  background-color: #d4d4d4;
  height: 1000vh;
`

const Shell = () => {
  return (
    <Background>
      <h1>SHELL</h1>
    </Background>
  )
}

export default Shell
