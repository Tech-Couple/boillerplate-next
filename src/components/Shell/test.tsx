import { render } from '@testing-library/react'

import Shell from '.'

describe('<Shell />', () => {
  it('should render the text', () => {
    const { container } = render(<Shell />)

    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render the colors correctly', () => {
    const { container } = render(<Shell />)

    expect(container.firstChild).toHaveStyle({ 'background-color': '#d4d4d4' })
  })
})
