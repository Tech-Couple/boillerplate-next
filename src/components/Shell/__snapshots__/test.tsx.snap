// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Shell /> should render the text 1`] = `
.c0 {
  background-color: #d4d4d4;
  height: 1000vh;
}

.c1 {
  color: #06092b;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    BOILERPLATE WITH: NEXTJS, TYPESCRIPT, JEST AND REACT TESTING LIBRARY, STYLED COMPONENTS AND STORYBOOK.
  </div>
</div>
`
